Integrations/Integration-Samanage.yml

```diff
@@ -0,0 +1,119 @@
+commonfields:
+  id: Samanage
+  version: -1
+name: Samanage
+display: Samanage
+category: Data Enrichment
```

Comment:
Probably. My current use case is data enrichment, but I have plans to add minimal case management in the future.